src/components/ConditionalFilter/API/getAPIOptions.tsx

```diff
@@ -0,0 +1,202 @@
+// @ts-strict-ignore
+import { ApolloClient } from "@apollo/client";
+import {
+  _GetAttributeChoicesDocument,
+  _GetCategoriesChoicesDocument,
+  _GetChannelOperandsDocument,
+  _GetCollectionsChoicesDocument,
+  _GetDynamicLeftOperandsDocument,
+  _GetProductTypesChoicesDocument,
+} from "@dashboard/graphql";
+
+import { FilterElement } from "../FilterElement";
+import { FilterContainer } from "../useFilterContainer";
+
+const getFilterElement = (value: any, index: number): FilterElement => {
+  const possibleFilterElement = value[index];
```

Comment:
suggestion: value is array or object maybe would be better to restrict type to Array<unknow> or Record<string, unknow>

Reply:
it just not yet typed, but you are right - there will be even more strict type